lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyUtils.java

```diff
@@ -88,17 +88,17 @@ public static URI getDocumentAsUri(DOMDocument document) {
      */
     public static Path findFileInWorkspace(String serverXmlURI, String filename) {
```

Comment:
I thought this one should be changed to take a `Path` as the second parameter also.